data/js/item/holidays/snowpile.js

```diff
@@ -57,6 +57,11 @@ function onUseChecked( pUser, iUsed )
 function onCallback0( socket, myTarget )
 {
 	var pUser = socket.currentChar;
+	if( pUser == null )
```

Comment:
Use ValidateObject( myObj ) (or the negative thereof) instead of comparing myObj to null (for characters and items)